### PR TITLE
OBSDOCS-977: Document improved scrape sample alerts

### PR DESCRIPTION
**Pull request by eromanova97**
Version(s): `enterprise-4.16`

Issue: [OBSDOCS-977](https://issues.redhat.com/browse/OBSDOCS-977)

Link to docs preview: [Creating scrape sample alerts
](https://76402--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/configuring-the-monitoring-stack.html#creating-scrape-sample-alerts_configuring-the-monitoring-stack)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by simonpasquier — COMMENTED** [review]
/lgtm

File: modules/monitoring-creating-scrape-sample-alerts.adoc [hunk 1]
```diff
@@ -71,3 +71,10 @@ spec:
 ----
 $ oc apply -f monitoring-stack-alerts.yaml
 ----
+
+. Additionally, you can check if a target has hit the configured limit:
+
+.. In the *Administrator* perspective of the web console, go to *Observe* -> *Targets* and select an endpoint with a `Down` status that you want to check.
+.. The *Scrape failed: sample limit exceeded* message is displayed if the endpoint failed because of an exceeded sample limit:
++
+image:monitoring-sample-limit-exceeded.png[Endpoint with exceeded sample limit]
```
Comment (adellape) [inline modules/monitoring-creating-scrape-sample-alerts.adoc:80]: Not sure if the screenshot is necessary, considering it's the only one (AFAICT) in the assembly and isn't providing any new information really. Also the screenshot here is from a dark mode web console, whereas docs.openshift and the Customer Portal don't have a built-in dark mode to match, so it's a little jarring.

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**